csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1144,6 +1151,29 @@ The namespace of the generic URI (`namespace`) of value type `string` with forma
 
 The URI (`uri`) of value type `string` with format `uri` contains the identifier itself.
 
+> These elements can be used to reference a specific component from a SBOM:
+
+*Example XYZ linking a component from a CycloneDX SBOM using the bomlink mechanism:*
+
+```
+          "x_generic_uris": [
+            {
+              "namespace": "https://cyclonedx.org/capabilities/bomlink/",
+              "uri": "urn:cdx:411dafd2-c29f-491a-97d7-e97de5bc2289/1#pkg:maven/org.jboss.logging/jboss-logging@3.4.1.Final?type=jar"
+            }
+```
+
+*Example XYZ linking a component from a SPDX SBOM:*
```

Comment:
Not sure if the "sounding" `S` of `SPDX` requires `a` &rightarrow; `an`...

Reply:
It is not consistent in the spec (both exist) but 48 time "an" outweighs 2 time "a". Same seems to apply for SBOM.

Reply:
Corrected in [3f7995d](https://github.com/oasis-tcs/csaf/pull/498/commits/3f7995d72ac866214353ba3795848f2547e3991a).